axelrod/strategies/calculator.py

```diff
@@ -10,6 +10,11 @@ class Calculator(Player):
     """
     Plays like (Hard) Joss for the first 20 rounds. If periodic behavior is
     detected, defect forever. Otherwise play TFT.
+
+
+    Names:
+
+    - Calculator: Original name by Marc Harper
```

Comment:
It's not mine -- it's in PRISON and likely predates it (it's in the file of classic strategies). Maybe one of the original Axelrod tournaments?